src/components/ClusterWizard.tsx

```diff
@@ -1,18 +1,19 @@
 import React, { FC, Fragment } from 'react';
 
-// import BaremetalInventory from './BaremetalInventory';
+import BaremetalInventory from './BaremetalInventory';
 import ClusterWizardNav from './ClusterWizardNav';
-// import ClusterWizardSteps from './ClusterWizardSteps';
 import CreateClusterForm from './createCluster/CreateClusterForm';
 
-const ClusterWizard: FC = (): JSX.Element => (
+interface ClusterWizardProps {
+  currentStepIndex: number;
```

Comment:
Can we still use a more concrete type to represent the wizard step?

E.g.:

```tsx
enum WizardStep {
    One,
    Two
}

let step: WizardStep = WizardStep.One;
```

It gives you much nicer comparisons:

```tsx
if (step === WizardStep.One) {
    
}
```

Instead of comparing against numbers.

Reply:
Done